src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -232,10 +234,24 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			if (this.cellModel.currentMode !== CellEditModes.WYSIWYG) {
 				needsTransform = false;
 			} else {
+				let result = undefined;
+				const isFile = !URI.parse(linkCalloutResult?.insertUnescapedLinkUrl).scheme.includes('http');
```

Comment:
There are other schemes possible too ('https' as one). Is there a reason why we can't check for 'file' directly?

Reply:
I remember that sometimes the scheme in Windows is 'c', instead of file; but I'm not sure if that only happens when trying to parse an absolute path ex: C://Users/hello/world, so maybe we can check only for 'file'.